engine/run_ensemble.py

```diff
@@ -234,33 +226,30 @@ def run_ensemble(
         append_job(job, job_list, parallel)
 
     # run the ensemble
-    for m_id in member_ids:
+    for member_id in member_ids:
 
-        Path(perturbed_run_dir.format(member_id=str(m_id))).mkdir(
+        Path(perturbed_run_dir.format(member_id=str(member_id))).mkdir(
             exist_ok=True, parents=True
         )
-        os.chdir(perturbed_run_dir.format(member_id=str(m_id)))
-
-        if member_type:
-            m_id_str = member_type + "_" + str(m_id)
-        else:
-            m_id_str = str(m_id)
+        os.chdir(perturbed_run_dir.format(member_id=str(member_id)))
 
         runscript = f"{run_dir}/{run_script_name}"
+        
+        complete_member_id = prepend_type_to_member_id(member_type, member_id)
```

Comment:
Just as a thought. Maybe `typed_memberd_id` or `prepended_member_id` would be more cleare what it means.